src/gdcdictionary/schemas/other_clinical_attribute.yaml

```diff
@@ -0,0 +1,1261 @@
+        - Arthritis
+        - Asthma
+        - Ataxia-Telangiectasia
+        - Atrial Fibrillation
+        - Autoimmune Lymphoproliferative Syndrome (ALPS)
+        - Avascular Necrosis
+        - Bacteroides fragilis
+        - Barrett's Esophagus
+        - Basal Cell Carcinoma
+        - Beckwith-Wiedemann
+        - Behcet's Disease
+        - Benign Thyroid Nodule
+        - Biliary Disorder
+        - Blood Clots
+        - Bone Fracture(s)
+        - Bronchitis
+        - Calcium Channel Blockers
+        - Cancer
+        - Cataracts
+        - Celiac Disease
+        - Cerebrovascular Disease
+        - Chlamydia
+        - Chloroma
+        - Cholelithiasis
+        - Chronic Fatigue Syndrome
+        - Chronic Pancreatitis
+        - Chronic Renal Failure
+        - Chronic Systemic Steroid Use
+        - Chronic Thyroiditis
+        - Cirrhosis, Unknown Etiology
+        - Clonal Hematopoiesis
+        - CNS Infection
+        - Colon Polyps
+        - Common Variable Immunodeficiency
+        - Congestive Heart Failure (CHF)
+        - Connective Tissue Disorder
+        - COPD
+        - Coronary Artery Disease
+        - Crohn's Disease
+        - Cryptococcal Meningitis
+        - Cryptogenic Organizing Pneumonia
+        - Cytomegalovirus (CMV)
+        - Deep Vein Thrombosis / Thromboembolism
+        - Denys-Drash Syndrome
+        - Depression
+        - Dermatomyosis
+        - Diabetes
+        - Diabetes, Type II
+        - Diabetic Neuropathy
+        - Diet Controlled Diabetes
+        - Diverticulitis
+        - Down Syndrome
+        - DVT/PE
+        - Dyslipidemia
+        - EBV Lymphoproliferation
+        - Eczema
+        - Epilepsy
+        - Epstein-Barr Virus
+        - Escherichia coli
+        - Familial Adenomatous Polyposis
+        - Fanconi Anemia
+        - Fibromyalgia
+        - Fibrosis
+        - Follicular Adenoma
+        - Gastritis
+        - Gastroesophageal Reflux Disease
+        - GERD
+        - Glaucoma
+        - Glycogen Storage Disease
+        - Goiter
+        - Gonadal Dysfunction
+        - Gorlin Syndrome
+        - Gout
+        - Graves' Disease
+        - Hashimoto's Thyroiditis
+        - Headache
+        - Heart Disease
+        - Helicobacter pylori Infection
+        - Hemihypertrophy
+        - Hemorrhagic Cystitis
+        - Hepatitis
+        - Hepatitis A Infection
+        - Hepatitis B Infection
+        - Hepatitis C Infection
+        - Hepatitis, Chronic
+        - Hereditary Non-Polyposis Colon Cancer
+        - Herpes
+        - Herpes Zoster
+        - High Grade Liver Dysplastic Nodule
+        - HIV/AIDS
+        - Hodgkin Lymphoma
+        - Human Papillomavirus Infection
+        - HUS/TTP
+        - Hypercalcemia
+        - Hypercholesterolemia
+        - Hyperglycemia
+        - Hyperlipidemia
+        - Hyperparathyroidism
+        - Hypertension
+        - Hyperthyroidism
+        - Hypospadias
+        - Hypothyroidism
+        - Hypothyroidism
```

Comment:
dupe?

Reply:
Yeah got added on accident since it was a new enum for risk_factors, but not comorbidities. Removing it.